aiidalab_qe/sssp.py

```diff
@@ -102,6 +102,7 @@ def _refresh_installed(self):
             self.set_trait("error", str(error))
             self.set_message(str(error))
         else:
+            self.set_trait("installed", True)
```

Comment:
To be safe I'd recommend:
```suggestion
            self.set_trait("installed", not bool(pseudos_to_install()))
```

Reply:
Good advice! If all the libraries are install by hands `bool(pseudos_to_install())` will be `False` empty list.